pom.xml

```diff
@@ -5,22 +5,184 @@
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-      <groupId>io.openshift.booster</groupId>
-      <artifactId>spring-boot-booster-parent</artifactId>
-      <version>1.5.13-2</version>
+      <groupId>io.openshift</groupId>
+      <artifactId>booster-parent</artifactId>
+      <version>22</version>
     </parent>
 
     <artifactId>spring-boot-cache-parent</artifactId>
     <version>1.5.13-3-SNAPSHOT</version>
     <packaging>pom</packaging>
 
-    <name>Spring Boot - Cache Booster</name>
+    <groupId>io.openshift.booster</groupId>
+  <name>Spring Boot - Cache Booster</name>
```

Comment:
Indentations are inconsistent through the whole file.

Reply:
This is the result of the updates being done in an automated fashion.
Once all the issues relating to functionality are fixed, I will go through the poms and manually fix such issues